applications/tari_base_node/src/recovery.rs

```diff
@@ -134,7 +134,8 @@ async fn do_recovery<D: BlockchainBackend + 'static>(
     // We dont care about the values, here, so we just use mock validators, and a mainnet CM.
     let rules = ConsensusManagerBuilder::new(NetworkType::LocalNet).build();
     let validators = Validators::new(MockValidator::new(true), MockValidator::new(true));
-    let temp_db_backend = BlockchainDatabase::new(temp_db, &rules, validators, BlockchainDatabaseConfig::default())?;
+    let temp_db_backend =
```

Comment:
This is fine, as this is not using its orphan pool at all.

Reply:
Agreed